lib/encryption.js

```diff
@@ -110,12 +111,77 @@ module.exports = function (ciphersAndHashes, crypto, promise, random) {
                 name: hmac.name
             },
             hmacKey: {
-                bytes: 32, // Arbitrary
+                // Arbitrary number of bytes
+                bytes: 32,
                 code: hmacKey.code,
                 iterations: Math.min(hmacIterations, 1000),
                 name: hmacKey.name
             }
+        };
+    }
+
+
+    /**
+     * Reads a header and generates the parameters from the data
+     *
+     * @param {Buffer} data
+     * @return {Promise.<encryption~header>}
+     */
+    function readHeaderAsync(data) {
```

Comment:
They might be, but order is now more dictated by the necessity of defining a variable or function before it is used.